.github/workflows/run-release.yml

```diff
@@ -409,6 +409,7 @@ jobs:
           - activiti-cloud
           - activiti-cloud-common-chart
           - activiti-cloud-full-chart
+          - activiti-scripts
```

Comment:
not sure if release for such project makes sense, found it due to behaviour of GH api

Reply:
Maybe it's better to have it, so we keep consistency between all there repositories